R/meta.R

```diff
@@ -308,6 +310,24 @@ make_meta_dfm <- function(inherit = NULL, ...) {
     update_meta(default, inherit, ...)
 }
 
+#' @rdname make_meta
+make_meta_fcm <- function(inherit = NULL, ...) {
+    if (is.null(inherit))
+        inherit <- list()
+    default <- list(
+        "unit" = "documents",
```

Comment:
What is your thought on object meta inherited from DFM or tokens? Currently only `unit` and `concatenator`?

Reply:
Sounds right. Maybe `what` as well?

Reply:
Added